labelbox/schema/id_type.py

```diff
@@ -1,7 +1,7 @@
-from enum import Enum
+from strenum import StrEnum
 
 
-class IdType(str, Enum):
+class IdType(StrEnum):
```

Comment:
https://blog.pecar.me/python-enum -> see this for anyone with why about this for 3.11 support